exodus_core/analysis/static_analysis.py

```diff
@@ -340,30 +348,30 @@ def save_icon(self, path):
         icon = self.get_icon_path()
         if icon is None:
             return None
-
         try:
             with zipfile.ZipFile(self.apk_path) as z:
                 with open(path, 'wb') as f:
                     f.write(z.read(icon))
                 _ = Image.open(path)
                 return path
         except:
-            logging.warning('Unable to get the icon from the APK - downloading from details')
+            logging.warning('Unable to get the icon from apk files downloading from details'.format(path))
             try:
                 saved_path = self._get_icon_from_details(path)
                 if os.path.isfile(path) and os.path.getsize(path) > 0:
                     logging.debug('Icon downloaded from Google Play')
                     return saved_path
             except Exception as e:
                 logging.error(e)
-                logging.warning('Unable to get the icon from details - downloading from GPlay')
+                logging.warning('Unable to get the icon from details - downloading from gplay')
```

Comment:
Could we just put "Google Play" directly ?

Reply:
Yes. It does not have any importance.

I changed the line while I was making test in my computer. It does not change nothing.